src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1038,18 +1038,15 @@ void goto_instrument_parse_optionst::instrument_goto_program()
     parse_function_pointer_restriction_options_from_cmdline(cmdline, options);
 
     if(
-      !options.get_list_option(RESTRICT_FUNCTION_POINTER_OPT).empty() ||
```

Comment:
:pick: please don't but a bunch of unrelated refactors into the same commit as it makes it harder to review

Reply:
And means you can't explain each one in the commit message

Reply:
Agreed. Will try to be more careful in the future.